docs/guide/walkthrough/goal.md

```diff
@@ -36,16 +36,28 @@ export class PingCommand extends Command {
 ```
 ### Using @sern/handler
 ```ts title="commands/ping.ts" showLineNumbers
-import { commandModule, CommandType } from '@sern/handler'
-import { publish } from '../plugins';
+import { commandModule, CommandType } from '@sern/handler';
+import { PermissionFlagsBits } from 'discord.js';
+
+export const config = {
+  dmPermission: true || false, //only choose one!
+	defaultMemberPermissions: [
+		PermissionFlagsBits.UseApplicationCommands
+	],
+  guildIds: ['id1', 'id2']
```

Comment:
change to `guildId1`, `...2`, just to clear things up and read less code when copying the snippet